profiler/README.md

```diff
@@ -5,10 +5,11 @@ It should be considered experimental and is completely unsupported.
 
 # configuration
 
-| name                                | default                | description                          |
-|-------------------------------------|------------------------|--------------------------------------|
-|`splunk.profiler.enabled`            | false                  | set to true to enable the profiler   |
-|`splunk.profiler.directory`          | "."                    | location of jfr files                |
-|`splunk.profiler.recording.duration` | 20                     | number of seconds per recording unit |
-|`splunk.profiler.keep-files`         | false                  | leave JFR files on disk id `true`    |
-|`splunk.profiler.logs-endpoint`      | http://localhost:4317  | where to send OTLP logs              |
+| name                                | default                | description                               |
+|-------------------------------------|------------------------|-------------------------------------------|
+|`splunk.profiler.enabled`            | false                  | set to true to enable the profiler        |
+|`splunk.profiler.directory`          | "."                    | location of jfr files                     |
+|`splunk.profiler.recording.duration` | 20                     | number of seconds per recording unit      |
+|`splunk.profiler.keep-files`         | false                  | leave JFR files on disk id `true`         |
+|`splunk.profiler.logs-endpoint`      | http://localhost:4317  | where to send OTLP logs                   |
+|`splunk.profiler.period.{eventName}` | n/a                    | customize period for a specific jfr event |
```

Comment:
should we document the most useful event name here? I assume that's "threaddump"?

Reply:
That makes sense.  It should also mention that it's milliseconds with no suffix, so I'll change it.  👍🏻